src/platform/utilities/data/index.js

```diff
@@ -14,6 +15,7 @@ import set from './set';
 import unset from './unset';
 
 export default {
+  buildQueryString,
```

Comment:
Please advise if I put this in the wrong place. I didn't realize until just now that these were `Lodash` replacement methods. Apologies!

Reply:
@rianfowler I removed this method altogether and used your suggestion to direct import `appendQuery` on both components. It was cleaner and did effectively the same thing.